src/components/OptionsSelector/BaseOptionsSelector.js

```diff
@@ -265,8 +271,9 @@ class BaseOptionsSelector extends Component {
         ) : <FullScreenLoadingIndicator />;
         return (
             <ArrowKeyFocusManager
+                disabledIndexes={this.disabledOptionsIndexes}
                 focusedIndex={this.state.focusedIndex}
-                maxIndex={this.props.canSelectMultipleOptions ? this.state.allOptions.length : this.state.allOptions.length - 1}
+                maxIndex={this.state.allOptions.length - 1}
```

Comment:
I am still not confident of the change, so we'll have to counter with the test so that nothing breaks.

Reply:
The purpose of original code was to have a state of no highlighted options when multiple options type, which is the 2nd issue of this GH (`the highlight goes out of the screen`)
So this change removes this state and doesn't break any others.

Only one possible regression will be:
https://github.com/Expensify/App/blob/dbfd0eb5ae2cd321db5075f088492f87ff1e8860/src/components/ArrowKeyFocusManager.js#L29-L31 (when there is only 1 option in list)
This case, that single option will not be highlighted by arrow keys in current codebase.
If it's fine, leave what it is now.
If it's issue, we can fix it by removing that code

Reply:
yeah lets fix it @vladnobenladen

Reply:
https://github.com/Expensify/App/pull/10477/commits/d44c976c99530797e3816cb51d4ee39157efb889

Reply:
Completely removing the code won't help.

Reply:
@mananjadhav any issue with it?

Reply:
Because we could pass any integer and it can cause if the index is negative. But we can ignore this change as I am also fixing it here https://github.com/Expensify/App/pull/7846#discussion_r957989921

Reply:
ok, makes sense in case user can pass any bad values like -1, -100 to `maxIndex` props.
added and pushed @mananjadhav